frontend/src/scenes/insights/Insights.stories.tsx

```diff
@@ -15,6 +15,18 @@ const meta: Meta = {
         },
         viewMode: 'story',
         mockDate: '2022-03-11',
+        viewport: {
```

Comment:
surely this can't make it worse

say the frustrated man whose been waiting for an hour to merge a posthog-js update 😡 

only, I would have expected more snapshots to be updated here to the new size... 🤔